### DISCUSSION TITLE
MWL-Modality Work List

### DISCUSSION THREAD

**Discussion by udara86** [dcm4che/dcm4che#852] [Q&A] [answered]
Is DCM4CHEE 2.18.X have Modality Work List (MWL) capability?

**Comment by gunterze** [comment, marked as answer]
Yes. But recommend to use dcm4chee-arc-5.x.

**Comment by udara86** [comment]
Thanks for the update. I look forward to it..


>>Best Regards From
  Maj Udara Pathirage
       *Master of IT, UCSC, Colombo | BSc in ENCM, Kelaniya | Dip. IT, NIBM
| MCS(SL) | MBCS CITP (UK)*
*        Reading for BEng (Hons) SE, UK *

*   Certifications:*
*           * AWS Certified Solutions Architect - Associate*
*           * AWS Certified Cloud Practitioner*
*           * SNOMED CT Certification*


*    Hotlines: 0718110497 | 0776021042*
